doc/Customization-guide/Documentation-contribution-and-style-guide/index.md

```diff
@@ -0,0 +1,245 @@
+---
+layout: page
+title: Documentation contribution and style guide
+permalink: /:path/
+nav: /:path/Customization-guide/Documentation-contribution-and-style-guide
+sort_idx: 80
+---
+{% assign root_url = page.url | split: '/'%}
+{% capture root_url  %} /{{root_url[1]}}/{{root_url[2]}}/{% endcapture %}
+
+- [Documentation style guide](#documentation-style-guide)
+  - [Sentences and grammar](#sentences-and-grammar)
+  - [Word choice](#word-choice)
+  - [Headings and capitalization](#headings-and-capitalization)
+  - [Technical terms and abbreviations](#technical-terms-and-abbreviations)
+  - [Font conventions](#font-conventions)
+  - [Code formatting](#code-formatting)
+  - [Page structure](#page-structure)
+  - [Markdown formatting](#markdown-formatting)
+  - [Jekyll formatting](#jekyll-formatting)
+- [Documentation contribution](#documentation-contribution)
+  - [Documentation preview](#documentation-preview)
+
+This guide introduces documentation conventions applicable to the Kaa project. The intent of this guide is to unify the look and feel of the documentation, make navigation predictable, usage and contribution simple.
+
+# Documentation style guide
+
+## Sentences and grammar
+* Use consistent terminology, so that every term means the same thing and is spelled in the same way throughout the entire documentation. If not properly introduced, synonyms can confuse the reader.
+* Give preference to present tense over past and future tenses; give preference to simple verb forms over perfect and continuous forms.
+* Use numbered lists for step-by-step procedures and bulleted list for an unordered set of items. Capitalize the first word of each list entry.
+* Use parallel sentence structures in bulleted and numbered lists.
+* End each entry of the list with a period if all the entries are complete sentences or if they are a mixture of fragments and complete sentences. If all entries are fragments, do not end them with periods or any other punctuation mark.
+* Use imperative mood in procedures, that is, formulate your instructions as giving commands to the user.
+* Avoid inventing new words or assigning words an unusual meaning.
+
+## Word choice
```

Comment:
Does this really need to be a separate paragraph?
I guess, this can be just an item within the "Sentences and grammar"